package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native",
-  "version": "0.12.0",
+  "version": "0.17.0-rc",
```

Comment:
Ah sure. But we need to change the contribution guide too clearly say it. That guide caused this.